com.Audentia.BlocNotes/com.Audentia.BlocNotes/DetailViewController.m

```diff
@@ -24,13 +29,45 @@ - (void)setDetailItem:(id)newDetailItem {
         [self configureView];
     }
 }
+- (IBAction)saveNote:(id)sender {
```

Comment:
ah ok, yes i will move the save content into its own method which the button or viewDidDisappear will call. I am modifying the save method to account for a "blank" new note. Also, I guess this sorta makes the save button redundant, but perhaps is reassuring to the user.